src/Shared/SidebarContainerCard/SidebarContainerCard.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+function SidebarContainerCard({ data }) {
+  return (
+    <>
+      <div className="sidebar--container">
+        <div className="sidebar--heading">{data[0].heading}</div>
+        <div className="sidebar--content">
+          <label htmlFor={data[0].type1}>
+            <input type="checkbox" id={data[0].type1} />
+            &nbsp;&nbsp;&nbsp;{data[0].input1}
```

Comment:
&nbsp; is. a non-line breaker and provide you empty space. You need to use only one in one line code.

Reply:
ok